web/angular-wallet/src/@fuse/services/match-media.service.ts

```diff
@@ -45,11 +45,13 @@ export class FuseMatchMediaService
                 distinctUntilChanged()
             )
             // @ts-ignore
-            .subscribe((change: MediaChange) => {
-                if ( this.activeMediaQuery !== change.mqAlias )
-                {
-                    this.activeMediaQuery = change.mqAlias;
-                    this.onMediaChange.next(change.mqAlias);
+            .subscribe((change: MediaChange[]) => {
```

Comment:
```suggestion
            .subscribe((change: MediaChange[] = []) => {
```
That way you could remove the if statement

Reply:
Would still need the if statement because we are assuming [0] exists... would still throw an error if not.

Reply:
ah yeah... you're right.... what a crappy API